test_remote.R

```diff
@@ -1,16 +1,145 @@
 
-setwd("~/git/gambl/") # set this path to point to your local clone of the gambl repo
 # in your config.yml file set repo_base under the remote config (at the top) to point to this path also
 library(GAMBLR)
+#skip this next line if you are running on the GSC network
 Sys.setenv(R_CONFIG_ACTIVE= "remote")
 
 
 config::get("project_base")
+# should point to an existing path on your local computer where you want the data to be stored
 #[1] "/Users/rmorin/gambl_results/"
+config::get("repo_base")
+# must refer to the location of the local clone of the gambl repo (morinlab/gambl)
+#[1] "/Users/rmorin/git/gambl/"
 
+# various tests
+# 1. get the metadata for genome and capture samples
 all_meta = get_gambl_metadata()
+all_meta = mutate(all_meta,pathology=ifelse(pathology=="BLL","BL",pathology))
+cap_meta = get_gambl_metadata(seq_type_filter = "capture")
 
-collated = collate_results(from_cache=TRUE)
+cap_meta = cap_meta %>% dplyr::filter(!sample_id %in% all_meta$sample_id)
 
+#load collated summary results
+collated = collate_results(from_cache=TRUE)
 collated = collate_results(from_cache=TRUE,seq_type_filter="capture")
+dim(collated)
+# this currently  prints out:
+# 2058 27
+
+#2. get mutations (assuming the files have been synced)
+# all these should proceed without error
+coding_ssm = get_coding_ssm(these_samples_metadata = all_meta,seq_type = "genome")
+
+cap_ssm = get_coding_ssm(these_samples_metadata = cap_meta,seq_type = "capture",seq_type_priority="genome")
+
+cap_ssm_hg38 = get_coding_ssm(these_samples_metadata = cap_meta,seq_type = "capture",projection = "hg38")
+
+
+dim(coding_ssm)
+# [1] 179807     45
+dim(cap_ssm)
+# [1] 626822     45
+
+which(coding_ssm$Tumor_Sample_Barcode %in% cap_ssm$Tumor_Sample_Barcode)
+# no overlap of IDs between the two because we used seq_type_priority
+
+all_ssm = bind_rows(coding_ssm,cap_ssm)
+
+
+blacklisted = GAMBLR:::annotate_ssm_blacklist(cap_ssm_hg38,seq_type = "capture",genome_build = "hg38")
+
+#test SV functions
+svar_all = get_combined_sv(min_vaf = 0.1,projection = "grch37")
+
+dim(svar_all)
+# [1] 450641     19
```

Comment:
This returned a very different value for me: 
```
[1] 150162     19
```

Reply:
I get 450641 on the gphost too. Must be a truncated file on your end.